### PR TITLE
Changes required to ensure spec is correct for developer site

### PR DESCRIPTION
**Pull request by franmoore05**
### What

Cosmetic changes to ensure the swagger spec displays correctly with any additional information on the developer site.

### How to review

Ensure the changes make sense.

### Who can review

Anyone.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -572,11 +561,12 @@ paths:
       tags:
       - "Public"
       summary: "Get observations for a census dataset"
-      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need.  If no geography, dimension, or options parameters are supplied then observations for the default geography are returned."
+      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need.  If no area-type parameters are supplied then observations for the default area-type are returned."
```
Comment (rafahop) [inline swagger.yaml:564]: Can we take the opportunity to remove the extra whitespace?
```suggestion
      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need. If no area-type parameters are supplied then observations for the default area-type are returned."
```

File: swagger.yaml [hunk 2]
```diff
@@ -44,9 +44,9 @@ parameters:
     required: true
     description: The unique filter output ID for a customised dataset
     in: path
-  geography:
-    description: "The name of the geography dimension that is being flexed, e.g. ward"
-    name: geography
+  area-type:
+    description: "The name of the area-type dimension that is being flexed, e.g. country"
```
[the remark this answers is not in the document]
Comment (franmoore05) [reply swagger.yaml:48]: It doesn't look like it's possible to do this in version 2.0 which we use for our swagger specs.

File: swagger.yaml [hunk 3]
```diff
@@ -170,8 +170,7 @@ paths:
       - $ref: '#/parameters/filter_output_id'
     get:
       tags:
-      - "Public"
-      - "Private"
+      - Public
```
[the remark this answers is not in the document]
Comment (franmoore05) [reply swagger.yaml]: This should now be amended.

File: swagger.yaml (hunk 1 above)
Comment (franmoore05) [reply swagger.yaml]: This has now been removed.

File: swagger.yaml [hunk 4]
```diff
@@ -572,11 +561,12 @@ paths:
       tags:
       - "Public"
       summary: "Get observations for a census dataset"
-      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need.  If no geography, dimension, or options parameters are supplied then observations for the default geography are returned."
+      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need.If no area-type parameters are supplied then observations for the default area-type are returned."
```
Comment (rafahop) [inline swagger.yaml:564]: It looks like we've gone from 2 whitespaces to none 🙈 
```suggestion
      description: "Get observations for a census dataset, with capability to flex and filter the data to the users need. If no area-type parameters are supplied then observations for the default area-type are returned."
```

File: swagger.yaml [hunk 5]
```diff
@@ -209,12 +208,11 @@ paths:
...
     post:
       tags:
-        - Public
-        - Private
+        - "Public"
       summary: Create a filter for a dataset
       description: >-
         Create a resource for listing a selection of dimensions and dimension
-        options to be added to filter for a dataset
+        options to be added to filter for a dataset.  Note - the is_area_type property is required to ensure a successful filter journey.
```
Comment (rafahop) [inline swagger.yaml:215]: Just notices an extra whitespace here too
```suggestion
        options to be added to filter for a dataset. Note - the is_area_type property is required to ensure a successful filter journey.
```

File: swagger.yaml (hunk 4 above)
Comment (franmoore05) [reply swagger.yaml]: One space added back in

File: swagger.yaml (hunk 5 above)
Comment (franmoore05) [reply swagger.yaml]: Space removed.

**Review by rafahop — APPROVED** [review]

**Pull request merged**